tests/test_resolvers/test_stack_output.py

```diff
@@ -50,6 +50,16 @@ def test_resolver(self, mock_get_output_value):
             sceptre_role="dependency_sceptre_role",
         )
 
+    @patch("sceptre.resolvers.stack_output.StackOutput._get_output_value")
+    def test_resolver__badly_formatted(self, mock_get_output_value):
+        stack = MagicMock(spec=Stack)
+        stack.name = "my/stack"
+
+        stack_output_resolver = StackOutput("not_a_valid_stack_output", stack)
```

Comment:
Does this simulate a stack config with invalid syntax?

Something like this:

```yaml
!stack_output not_a_valid_stack_output
```

Reply:
@iainelder well both are examples of things that aren't valid stack outputs and both would activate the bug reported in #1388  But I think it's a bit more readable and probably more correct as-is? (Since it's the whole field here that is "not a valid stack output" and not merely the argument passed to the resovler.)

Reply:
Yes, I'm not suggesting a change here. I'm just trying to relate these Python internals to my experience of using the feature via YAML. I understand now that the StackOutput class in Python implements the `!stack_output` function in YAML.